torch/jit/_recursive.py

```diff
@@ -29,6 +31,16 @@
     "dump_patches",
 ]
 
+ignored_properties = [
+    # Temporary fix for RNN module property named 'all_weights' being scripted
+    "all_weights",
```

Comment:
Can we switch this instead to a list of (Python Class, [ignored_properties]) tuples, and we ignore a property if the scripted class is a subclass of our registered class ? right now we're ignoring these names indiscriminately

Reply:
Fixed. However, I had to use `torch.nn.Module` as the class for all of the `RecursiveScriptModule` properties that we need to ignore because of circular import problems.